commands/user_e2e_test.go

```diff
@@ -136,3 +136,62 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestListUserCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	// populate map for checking
+	userPool := []string{
+		s.th.BasicUser.Username,
+		s.th.BasicUser2.Username,
+		s.th.TeamAdminUser.Username,
+		s.th.SystemAdminUser.Username,
+	}
+	for i := 0; i < 10; i++ {
+		usr := s.th.CreateUser()
```

Comment:
nit: just to make sure that user creation goes well, could you please add an error check here?

```suggestion
		usr, err := s.th.CreateUser()
		s.Require().Nil(err)
```